addons/mail/static/src/models/attachment_viewer/attachment_viewer.js

```diff
@@ -20,6 +22,71 @@ function factory(dependencies) {
                 dialog.delete();
             }
         }
+
+        /**
+         * Handle onclick on next button
```

Comment:
onclick -> click (just the name of the event itself, not with the "on" prefix)
dot at the end
also guideline in jsdoc is to have the first word be 3rd person singular (Handles)

check all of them for these